posts/2024-03-08-Rustup-1.27.0.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Announcing Rustup 1.27.0"
+author: The Rustup Working Group
+---
+
+The rustup working group is happy to announce the release of rustup version 1.27.0.
```

Comment:
Rustup was changed to a team a couple years ago.

```suggestion
author: The Rustup Team
---

The rustup team is happy to announce the release of rustup version 1.27.0.
```

Reply:
@ehuss Yes you're right! We're still called `wg-rustup` on Discord somehow but we should definitely change that 🤦‍♀️